deck-of-cards-kata-solutions/src/main/java/bnymellon/codekatas/deckofcards/Rank.java

```diff
@@ -42,6 +42,11 @@
     @Override
     public String toString()
     {
-        return emoji;
+        return this.emoji;
+    }
+
+    public Card of(Suit suit)
```

Comment:
any specific reason you added this method?

Reply:
I want to be able to create a Card in the code the same way I would say it. For example, "Ace of Clubs" in code can be `Rank.ACE.of(Suit.CLUBS)` instead of `new Card(Rank.ACE, Suit.CLUBS)`. This means we can use `Rank::of` instead of `Card::new` when creating a Deck of Cards using `cartesianProduct`.